core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -658,13 +658,13 @@ public function create()
 	 * @param string $source
 	 *
 	 * @throws AccessDeniedException
-	 * @throws InternalServerErrorException
+	 * @throws BadRequestException
 	 */
 	public function cut($source=null)
 	{
 		if ($GLOBALS['TL_DCA'][$this->strTable]['config']['notSortable'] ?? null)
 		{
-			throw new InternalServerErrorException('Table "' . $this->strTable . '" is not sortable.');
+			throw new AccessDeniedException('Table "' . $this->strTable . '" is not sortable.');
```

Comment:
Wouldn’t a `BadRequestException` be a better fit here?
There is no way to gain access, even an admin user would get this exception so “access denied” seems wrong to me.

Reply:
Probably also 422? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422

Reply:
To me, the `notEditable` and `notSortable` are permission settings. We often set them in the `checkPermission` callbacks. And we might migrate them to voters later on.